blenderproc/python/renderer/RendererUtility.py

```diff
@@ -655,6 +677,10 @@ def render(output_dir: Optional[str] = None, file_prefix: str = "rgb_", output_k
         load_keys = {'colors', 'distance', 'normals', 'diffuse', 'depth', 'segmap'}
         keys_with_alpha_channel = {'colors'} if bpy.context.scene.render.film_transparent else None
 
+    if view_transform not in {"AgX", "Standard", "Filmic", "Filmic Log", "False Color", "Raw"}:
+        raise ValueError(f"Unknown view transform {view_transform}")
+    
+    bpy.context.scene.view_settings.view_transform = view_transform
```

Comment:
Can we move the new `view_transform` parameter outside of the render function and put it into the `set_output_format` function? Then you could set the default parameter here in the init class https://github.com/DLR-RM/BlenderProc/blob/95ba9f3e77ae788662d76e5625d427d00e73a3c3/blenderproc/python/utility/Initializer.py#L138C25-L138C42

Reply:
Done.